public/app/percona/add-instance/components/AddInstance/AddInstance.messages.ts

```diff
@@ -1,6 +1,6 @@
 export const Messages = {
   titles: {
-    rds: 'AWS RDS MySQL or Aurora MySQL',
+    rds: 'AWS RDS or Aurora',
```

Comment:
```suggestion
    rds: 'Amazon RDS',
```

From AWS documentation:
 
![Screenshot 2021-12-09 at 11 07 22](https://user-images.githubusercontent.com/16366502/145385508-d58c37ba-6bc2-4c59-9e3a-e46e954fafb4.png)

Reply:
thanks!